### PR TITLE
Feature/gg 68 update photo registry

### PR DESCRIPTION
**Pull request by curlyjay**

### CODE REVIEW

**Comment by curlyjay** [comment]
Each time an existing employee is recognized, the new code will reference the last time stamp that they were scanned and compare it to the specified time delay variable. FaceID will recognize any face, but only update new frames of that face once the time delay passes. 

To test, run the Django server first, then the DB_faceID.py program, and select option 2 to run face recognition. Make sure employees are created in the web interface and faceID program.

File: FaceID/DB_faceID.py [hunk 1]
```diff
@@ -280,16 +303,72 @@ def searchFace():
...
+
+                    
+                    
+                    for person in photoRegister:
+                        extractEmpID = (person[1].split("_"))[0]
+
+                        if (int(extractEmpID) == EmpID):
+
+
+
```
Comment (kesslerk) [inline FaceID/DB_faceID.py]: Can you remove some of these extra new lines. There is usually no need to have more than one maybe two new lines

Comment (curlyjay) [reply FaceID/DB_faceID.py]: I went through and fixed the spacing

File: FaceID/DB_faceID.py [hunk 2]
```diff
@@ -280,16 +294,66 @@ def searchFace():
...
+                            timeNow = (str(currentTime.hour) + ":" + str(currentTime.minute) + ":" + str(currentTime.second)).split(":")
+                            timeNow = list(map(int,timeNow))
+
+                            hourDiff = timeNow[0] - lastScan[0]
+                            minDiff = timeNow[1] - lastScan[1]
+                            secDiff = timeNow[2] - lastScan[2]
+                            
+                            if hourDiff<0: hourDiff += 24
+                            if minDiff<0: minDiff += 60
+                            if secDiff<0: secDiff += 60
```
Comment (kesslerk) [inline FaceID/DB_faceID.py]: These three line should have spaces around the `<` for consistency and readability

**Comment by orndorffc** [comment]
Tested & ready to merge from an accuracy standpoint @kesslerk

**Review by orndorffc — APPROVED** [review]
@kesslerk Reviewed for accuracy and is ready for merge

**Comment by kemptonk** [comment]
The stored times need to be changed to dates.

File: FaceID/DB_faceID.py (hunk 2 above)
Comment (curlyjay) [reply FaceID/DB_faceID.py]: Fixed spacing and date threshold management

**Pull request merged**